oxidation/crates/parsec_core_fs/src/storage/mod.rs

```diff
@@ -0,0 +1,12 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+mod chunk_storage;
+mod local_database;
+mod manifest_storage;
+#[allow(dead_code)]
```

Comment:
why ?

Reply:
Because I don't use user_storage for the moment

Reply:
then you should add a comment ;-)

Reply:
Ok !